3rdparty/ippicv/downloader.cmake

```diff
@@ -13,7 +13,7 @@ function(_icv_downloader)
     set(OPENCV_ICV_PLATFORM "macosx")
     set(OPENCV_ICV_PACKAGE_SUBDIR "/ippicv_osx")
   elseif(UNIX)
-    if(ANDROID AND (NOT ANDROID_ABI STREQUAL x86))
+    if(ANDROID AND (NOT CMAKE_ANDROID_ARCH STREQUAL x86))
```

Comment:
Should be `CMAKE_ANDROID_ARCH_ABI`, see:
* http://polly.readthedocs.io/en/latest/toolchains/android/old.html